src/main/java/com/kkm/kkm_server_v2/domain/post/service/FindByCategoryPostService.java

```diff
@@ -24,7 +24,12 @@ public class FindByCategoryPostService {
     @Transactional(readOnly = true)
     public PostListResponse execute(int page, int size, PostCategory category) {
         Pageable pageable = PageRequest.of(page - 1, size, Sort.Direction.DESC, "pullDate");
-        Page<Post> list = postRepository.findByCategory(category, pageable);
+        Page<Post> list;
+        if (category == PostCategory.ALL) {
+            list = postRepository.findAll(pageable);
+        } else {
+            list = postRepository.findByCategory(category, pageable);
+        }
```

Comment:
if 문 말고 삼항 연산자로 사용하는 거는 어떤가요? 별로면 말고 ㅎㅎ
```suggestion
        Page<Post> list = category == PostCategory.ALL ? postRepository.findAll(pageable) : postRepository.findByCategory(category, pageable);
```